### PR TITLE
Conversion from kPa to PSI

### PR DESCRIPTION
**Pull request by anant00729**
To Test,

1. add some sensors to the farm and make sure you have the sensor readings
2. long click on the sensor to see the temperature and soil water potential readings 
3. go to http://localhost:3000/profile and change metric to imperial 
4. come back to the farm page 
5. long press and check if you can see the value of soil water potential is in PSI

### CODE REVIEW

File: packages/webapp/src/components/Map/PreviewPopup/utils.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-const metric = ['metric', 'celsius'];
-const imperial = ['imperial', 'fahrenheit'];
+const metric = ['metric', 'celsius', 'kpa'];
+const imperial = ['imperial', 'fahrenheit', 'psi'];
```
Comment (JimmyRowland) [inline packages/webapp/src/components/Map/PreviewPopup/utils.js:3]: temperature is already in convert-unit library
https://www.npmjs.com/package/convert-units#temperature
Also, the library support custom units
https://github.com/convert-units/convert-units

**Review by JimmyRowland — CHANGES_REQUESTED** [review]

File: packages/webapp/src/components/Map/PreviewPopup/index.jsx [hunk 2]
```diff
@@ -55,7 +66,7 @@ export default function PurePreviewPopup({ location, history, sensorReadings, st
   const classes = useStyles();
   const { t } = useTranslation();
   const { units } = useSelector(userFarmSelector);
-  const { reading_types } = useSelector(sensorReadingTypesByLocationSelector(location.id));
+  const { reading_types = [] } = useSelector(sensorReadingTypesByLocationSelector(location.id));
```
Comment (JimmyRowland) [inline packages/webapp/src/components/Map/PreviewPopup/index.jsx:69]: why it's location.id not location.location_id

File: packages/webapp/src/components/Map/PreviewPopup/utils.js (hunk 1 above)
Comment (anant00729) [reply packages/webapp/src/components/Map/PreviewPopup/utils.js:3]: The implementation of temperature conversion was added by sein. I have just worked on the soil water potential

File: packages/webapp/src/components/Map/PreviewPopup/index.jsx (hunk 2 above)
Comment (anant00729) [reply packages/webapp/src/components/Map/PreviewPopup/index.jsx:69]: This was added by sein. I don't know why the variable name is kept as id

**Review by wasswarichard — APPROVED** [review]
looks fine

**Pull request closed without merging**